django/db/backends/sqlite3/creation.py

```diff
@@ -4,11 +4,15 @@
 
 from django.core.exceptions import ImproperlyConfigured
 from django.db.backends.base.creation import BaseDatabaseCreation
+from django.utils.encoding import force_text
 from django.utils.six.moves import input
 
 
 class DatabaseCreation(BaseDatabaseCreation):
 
+    def is_database_in_memory(self, database_name):
```

Comment:
Sounds good, I'll make it static then. For future reference, as one data point, here's one case where I would have considered subclassing the `DatabaseCreation` class (but for PostgreSQL, if the ticket hadn't been accepted): https://github.com/django/django/pull/7092